static_sources/eateryAttributes.json

```diff
@@ -0,0 +1,67 @@
+{
+  "104-West": {
+    "reserveLink": "https://www.opentable.com/restaurant/profile/1080703"
+  },
+  "Bear-Necessities": {
+    "exceptions": [
+      "Mobile Order Only"
+    ],
+    "isGet": true
+  },
+  "Becker-House-Dining": {
+    "reserveLink": "https://www.opentable.com/restaurant/profile/1080700"
+  },
+  "Bug-Stop-Bagels": {
+    "isGet": true
+  },
+  "Cafe-Jennie": {
+    "isGet": true
+  },
+  "Carols-Cafe": {
+    "isGet": true
+  },
+  "Cook-House-Dining": {
+    "reserveLink": "https://www.opentable.com/restaurant/profile/1080697"
+  },
+  "Cornell-Dairy-Bar": {
+    "isGet": true
+  },
+  "Goldies-Cafe": {
```

Comment:
do these `Cafe`s not need accent marks on the e?

Reply:
nope, slugs remove any weird characters